src/App.js

```diff
@@ -8,7 +8,7 @@ import Categories from './Components/Categories';
 
 function App() {
   return (
```

Comment:
- [ ] Your app should be identical to the provided styling, kindly remove the outer space with the background so the app can take the whole space

|Your design|How it should be|
|-|-|
|![Screenshot_20220716_171436](https://user-images.githubusercontent.com/25227022/179363210-f6937d4d-6a61-4ead-9eb5-d82fea519e80.png)|![Screenshot_20220716_171504](https://user-images.githubusercontent.com/25227022/179363217-396b3edd-dd99-40ea-b2b1-f365352039ca.png)|